bundles/org.openhab.ui/web/src/components/developer/developer-sidebar.vue

```diff
@@ -493,6 +494,9 @@ export default {
         if (m.configuration.type && m.configuration.type.toLowerCase() === query) {
           return true
         }
+        if (query.indexOf('block') >= 0 && m.configuration.blockSource !== undefined) {
```

Comment:
```suggestion
        if (query.indexOf('block') === 0 && m.configuration.blockSource !== undefined) {
```

This is enough to match `block`, `blockly`, `blockSource` and avoids matching search terms like `oh-block`.

We could even go a step further and only match these three terms above:

```suggestion
        if (['block', 'blockly', 'blocksource'].includes(query) && m.configuration.blockSource !== undefined) {
```

Reply:
Thanks. Adjusted.